pkg/simple/client/monitoring/metricsserver/metricsserver_test.go

```diff
@@ -213,32 +213,36 @@ func TestGetNamedMetrics(t *testing.T) {
 		},
 	}
 	podMetricsTests := []struct {
-		metrics       []string
-		filter        string
-		expected      string
-		podName       string
-		namespaceName string
+		metrics          []string
+		filter           string
+		namespacedFilter string
+		expected         string
+		podName          string
+		namespaceName    string
 	}{
 		{
-			metrics:       []string{"pod_cpu_usage", "pod_memory_usage_wo_cache"},
-			filter:        "pod1$",
-			expected:      "metrics-vector-4.json",
-			podName:       "",
-			namespaceName: "",
+			metrics:          []string{"pod_cpu_usage", "pod_memory_usage_wo_cache"},
+			filter:           "pod1$",
+			namespacedFilter: "",
+			expected:         "metrics-vector-4.json",
+			podName:          "",
+			namespaceName:    "",
 		},
 		{
-			metrics:       []string{"pod_cpu_usage", "pod_memory_usage_wo_cache"},
-			filter:        "default/pod2$",
-			expected:      "metrics-vector-5.json",
-			podName:       "",
-			namespaceName: "",
+			metrics:          []string{"pod_cpu_usage", "pod_memory_usage_wo_cache"},
+			filter:           "",
+			namespacedFilter: "default/pod2$",
```

Comment:
Should we add more entries here in the test? because `namespacedFilter` could be like this `default/pod2|default/pod3|default/pod4`

Reply:
Fixed